tools/install.sh

```diff
@@ -163,6 +163,9 @@ mkdir -p ../project ../project/libraries ../project/instruments ../project/templ
 # Setting 777 permissions for templates_c
 chmod 777 ../smarty/templates_c
 
+# Changing group to www-data to give permission to create directories in Document Repository module
+sudo chown www-data.www-data ../modules/document_repository/user_uploads
```

Comment:
Should the user/group maybe be a shell variable? Then here (and other places in the install script that do similar things), we can detect the OS and set it to the appropriate username/group instead of assuming ubuntu..